examples/esm/package.json

```diff
@@ -0,0 +1,14 @@
+{
+  "private": true,
```

Comment:
Do we want folks to use this example? If so, we'll need a README here.

Reply:
Sure, I'll add README